src/components/ToastNotification/ToastNotification.tsx

```diff
@@ -62,22 +64,37 @@ export const ToastNotification = ({
     className,
   );
 
+  // if both are defined, temporarily prefer the original (misspelled) prop for now, but throw a warning
+  const tempDismissType =
+    dissmissType && dismissType ? dissmissType : dismissType;
+
   assertEdsUsage(
-    [!!timeout && typeof onDismiss === 'undefined' && dissmissType === 'auto'],
+    [
+      !!timeout &&
+        typeof onDismiss === 'undefined' &&
+        tempDismissType === 'auto',
+    ],
     'When using dismissType=auto, an onDismiss method must be defined',
     'error',
   );
 
+  // TODO(next-major): remove this misspelled prop at next major release
+  assertEdsUsage(
```

Comment:
The assert is good, but should we also add a `@deprecated` flag to the prop so that developers can more easily catch it without needing to run the code?

Reply:
Yah good call. The old deprecations were cleaned up at the last major, so didn't think to add one!